apps/web/vibes/soul/components/button/index.tsx

```diff
@@ -5,58 +5,71 @@ import { ReactNode, Ref, forwardRef } from 'react'
 
 import clsx from 'clsx'
 
+import { useBrandContext } from '@/components/preview/brand-context'
+import getBrandShade from '@/vibes/soul/getBrandShade'
+
 export interface Props {
   className?: string
   link?: { href: string; target?: string }
-  variant?: 'primary' | 'secondary'
+  variant?: 'primary' | 'secondary' | 'tertiary'
   size?: 'default' | 'small'
   children?: ReactNode
-  borderGlow?: boolean
 }
 
 export const Button = forwardRef(function Button(
-  {
-    className,
-    link,
-    variant = 'primary',
-    size = 'default',
-    children = 'Button',
-    borderGlow = true,
-  }: Props,
+  { className, link, variant = 'primary', size = 'default', children = 'Button' }: Props,
   ref: Ref<HTMLAnchorElement>
 ) {
+  const { activeBrand } = useBrandContext()
```

Comment:
This is interesting, it will make some components client components even when they only need style. 🤔

Reply:
Good catch! This method of styling has been removed and replaced by a css only solution.